Automattic/wp-calypso
# CRO i5: Add new plans filter bar.

### Changes proposed in this Pull Request

This PR add the new CRO i5 iteration of the plans page filter bar for Calypso and Jetpack Cloud.

Fixes 1196341175636977-as-1199149328812588

#### Implementation notes:

- I did not implement the new scroll behavior is this PR (only pin the plans filter bar. The page heading and navigation should not also be pinned but rather disappear when scrolled)

### Testing instructions

- Download the PR
- Run both Calypso and Jetpack cloud concurrently
- Select the `i5 - Saas table design` variant of the a/b test jetpackConversionRateOptimization
- Visit the Plans page
- In both Calypso and Jetpack Cloud, check:
    - That the plans filter bar matches the Figma design (In desktop & mobile)
    - That the Filter Bar duration toggle functions as it should (Changes the plans/products billing duration and prices)
    - Select the previous a/b variant (v2 - slide outs) and check that the previous plans filter bar iteration was not affected.

### Screenshots

### Figma designs

Desktop
![Screenshot on 2020-11-11 at 08-44-49](https://user-images.githubusercontent.com/11078128/98840687-f3dc5a00-23fb-11eb-8544-955b15dfd254.png)

Desktop scrolled
![Screenshot on 2020-11-11 at 08-45-55](https://user-images.githubusercontent.com/11078128/98840518-c2fc2500-23fb-11eb-92be-c4f67e8d71c6.png)

Mobile
![Screenshot on 2020-11-11 at 08-47-14](https://user-images.githubusercontent.com/11078128/98840535-c68fac00-23fb-11eb-97db-248dce90f35d.png)

Mobile scrolled
![Screenshot on 2020-11-11 at 08-47-53](https://user-images.githubusercontent.com/11078128/98840596-d6a78b80-23fb-11eb-9089-2e0b9765df81.png)

.
.
### This PR screenshots:

Desktop
![Screenshot on 2020-11-11 at 08-48-27](https://user-images.githubusercontent.com/11078128/98840751-0c4c7480-23fc-11eb-8be6-043d8ba636ea.png)

Desktop scrolled
![Screenshot on 2020-11-11 at 08-49-11](https://user-images.githubusercontent.com/11078128/98840757-0eaece80-23fc-11eb-9a78-02381d4f97be.png)

Mobile
![Screenshot on 2020-11-11 at 08-49-52](https://user-images.githubusercontent.com/11078128/98840765-10789200-23fc-11eb-9a73-562c88bcfe69.png)

Mobile scrolled
![Screenshot on 2020-11-11 at 08-50-25](https://user-images.githubusercontent.com/11078128/98840774-12daec00-23fc-11eb-9098-85188e7ee19b.png)
